src/2_resources.js

```diff
@@ -93,7 +93,8 @@ resources.open = {
 	endpoint: "/v1/open",
 	method: utils.httpMethod.POST,
 	params: {
-		"browser_fingerprint_id": validator(true, branch_id),
+		"browser_fingerprint_id": validator(false, branch_id),
+		"alternative_browser_fingerprint_id": validator(false, branch_id),
```

Comment:
alternative_browser_fingerprint_id looks exactly the same as browser_fingerprint_id. Shouldn't it be the value from storage instead based on the ticket desc?